TestModels/SimpleTypes/SimpleInteger/runtimes/rust/src/client.rs

```diff
@@ -2,14 +2,9 @@
 
 use aws_smithy_types::error::operation::BuildError;
 
-#[derive(Debug)]
-pub(crate) struct Handle {
-    pub(crate) inner: ::dafny_runtime::Object<dyn ::simple_integer_dafny::r#_simple_dtypes_dinteger_dinternaldafny_dtypes::ISimpleTypesIntegerClient>
-}
-
 #[derive(::std::clone::Clone, ::std::fmt::Debug)]
 pub struct Client {
-    handle: ::std::sync::Arc<Handle>,
+    pub(crate) client: ::dafny_runtime::Object<dyn ::simple_integer_dafny::r#_simple_dtypes_dinteger_dinternaldafny_dtypes::ISimpleTypesIntegerClient>
```

Comment:
Nit picky, but how about `dafny_client` instead? Only because `client.client....` is a bit weird/confusing elsewhere.